setup.py

```diff
@@ -0,0 +1,15 @@
+from setuptools import setup, find_packages
+from sys import stderr
+
+
+version = '0.0.1'
+
+print('Hi!', file=stderr)
+print(find_packages(exclude=['ez_setup', 'tests', 'predicates']), file=stderr)
+
+setup(name='PythonVoronoi', # Feel free to change this
+      version=version,
+      author='Drew Monroe', # feel free to change this, too
```

Comment:
You guys should all be in on this too, not just me! Just because it's in my repo doesn't mean it's my project, you guys deserve credit too :) Also throw Don's name in here?